utils/recaptcha/recaptcha.ts

```diff
@@ -3,14 +3,14 @@ import { RECAPTCHA_REQUEST_KEY, RECAPTCHA_SITE_KEY, RECATCHA_ACTIONS } from "con
 import { httpClient } from "../../configs/api/httpClient/HttpClient";
 
 export const recaptchaVerify = async () => {
-  const grecaptcha = (window as any)?.grecaptcha;
+  const grecaptcha = window.grecaptcha;
   const token = await grecaptcha.execute(RECAPTCHA_SITE_KEY, { action: RECATCHA_ACTIONS.SUBMIT });
   const res = await httpClient.post(`${process.env.NEXT_PUBLIC_APP_URL}${RECAPTCHA_ROUTE}`, {
     [RECAPTCHA_REQUEST_KEY]: token
   });
-  // eslint-disable-next-line no-console
-  console.log(res, "result");
-  if (token) {
+  if (res?.data?.success) {
     return true;
+  } else {
+    return false;
```

Comment:
```suggestion
  return res?.data?.success
```

`res?.data?.success` is either true or false | undefined.